nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/HashAttribute.java

```diff
@@ -51,6 +50,9 @@
 import org.apache.nifi.processor.util.StandardValidators;
 
 /**
```

Comment:
It is too bad there isn't a nifi deprecated annotation, or a way for the system to pick up the standard deprecated annotation and give a visual clue in the ui

Reply:
Yeah, I'll see about adding that